core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -243,7 +243,13 @@
   GENERAL_MAX_SCANNER_RETRY_PERIOD("general.max.scanner.retry.period", "5s",
       PropertyType.TIMEDURATION,
       "The maximum amount of time that a Scanner should wait before retrying a failed RPC"),
-
+  // experimental - micrometer properties
+  @Experimental
+  GENERAL_METRICS_CONFIGURATION_PROPERTIES_FILE("general.metrics.configuration.properties",
+      "accumulo.metrics.configuration.properties", PropertyType.STRING,
+      "The Accumulo metrics configuration properties filename. The file is used to"
+          + " configure micrometer MeterRegistries that loaded at runtime using classes that"
+          + " extend MetricsServiceLoader.class"),
```

Comment:
After the most recent changes in #2190, properties will need `since` information. I figured I would leave a comment regarding it.

```suggestion
  @Experimental
  GENERAL_METRICS_CONFIGURATION_PROPERTIES_FILE("general.metrics.configuration.properties",
      "accumulo.metrics.configuration.properties", PropertyType.STRING,
      "The Accumulo metrics configuration properties filename. The file is used to"
          + " configure micrometer MeterRegistries that loaded at runtime using classes that"
          + " extend MetricsServiceLoader.class", "2.1.0"),
```

Reply:
thanks!